### ISSUE TITLE
Train

### ISSUE THREAD

**Issue by wuhucheng** [ggsDing/SCanNet#4]
I am training myself with the training code you provided and the evaluation metrics cannot achieve the results in your article, what do I need to do?
![1](https://github.com/ggsDing/SCanNet/assets/45914165/28acd808-e9d6-47eb-b9cc-d5127a616ae7)

**Comment by ggsDing** [comment]
This is abnormal. Did you change the batch size or other settings?
Btw, the trained weights have been uploaded. Also the train/val split is provided in 'datasets' folder. You are welcome to verify the accuracy,

**Comment by ggsDing** [comment]
Sorry I just checked the code and the batch size is too small. I just changed that to '8'.

**Comment by wuhucheng** [comment]
I organized the data according to the divisions you provided and retrained it.
![图片](https://github.com/ggsDing/SCanNet/assets/45914165/4bd9ecf6-8df3-4c0e-a25b-81a5599de0a8)
Also, can you provide a comparison test algorithm for these above? 1499549355@qq.com十分感谢您

**Comment by ggsDing** [comment]
> I organized the data according to the divisions you provided and retrained it. ![图片](https://user-images.githubusercontent.com/45914165/251691468-4bd9ecf6-8df3-4c0e-a25b-81a5599de0a8.png) Also, can you provide a comparison test algorithm for these above? [1499549355@qq.com](mailto:1499549355@qq.com)十分感谢您

These are not the original implementations and have been changed to fit the SCD settings. Therefore, I do not think it is quite appropriate to provide them. You can ask the authors for their original implementations or reimplement them.
